x/bank/client/cli/encode.go

```diff
@@ -0,0 +1,76 @@
+package cli
```

Comment:
Both this and the rest endpoint should live under the `x/auth` package. The same applies to the `broadcast` command/endpoint too, though I reckon we can get this in as it is since such refactoring would be out of context of the PR. /cc @jackzampolin @cwgoes @alexanderbez

Reply:
Yes, both should be moved to the auth module. Preferably in this PR.

Reply:
Cool, I've moved both to the auth module.